CMakeLists.txt

```diff
@@ -149,7 +150,14 @@ if(NOT CPPUNIT_FOUND)
     message(FATAL_ERROR "CppUnit required to compile sattools")
 endif()
 
-find_package(ZeroMQ)
+########################################################################
+# Option to include zmq block
+########################################################################
+option(INCLUDE_ZMQ_BLOCK "Include blocks that need ZMQ" ON)
```

Comment:
Let's use gnuradio conventions, `ENABLE_SATTOOLS_ZEROMQ`, similar to GR_ZEROMQ

https://github.com/gnuradio/gnuradio/blob/master/gr-zeromq/CMakeLists.txt